File: app/src/main/java/br/com/wcisang/alojinha/model/Product.java
package br.com.wcisang.alojinha.model;

/**
 * Created by WCisang on 07/06/2018.
 */
public class Product {

    private int id;
    private String descricao;
    private String nome;
    private float precoDe;
    private float precoPor;
    private String urlImagem;
    private Category categoria;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public float getPrecoDe() {
        return precoDe;
    }

    public void setPrecoDe(float precoDe) {
        this.precoDe = precoDe;
    }

    public float getPrecoPor() {
        return precoPor;
    }

    public void setPrecoPor(float precoPor) {
        this.precoPor = precoPor;
    }

    public String getUrlImagem() {
        return urlImagem;
    }

    public void setUrlImagem(String urlImagem) {
        this.urlImagem = urlImagem;
    }

    public Category getCategoria() {
        return categoria;
    }

    public void setCategoria(Category categoria) {
        this.categoria = categoria;
    }
}

File: app/src/main/java/br/com/b2w/lodjinha/features/product/Product.kt
package br.com.b2w.lodjinha.features.product

import br.com.b2w.lodjinha.features.category.Category
import com.google.gson.annotations.SerializedName

data class Product(
    val id: Int,
    val urlImagem: String,
    @SerializedName("categoria") val category: Category,
    @SerializedName("descricao") val description: String,
    @SerializedName("nome") val name: String,
    @SerializedName("precoDe") val oldPrice: Double,
    @SerializedName("precoPor") val newPrice: Double
)